docs/reference/api/client/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Client (Kn)"
 linkTitle: "Client (Kn)"
-weight: 10
+weight: 30
 type: "docs"
 ---
```

Comment:
Global: For each of the files you are moving, there needs to be a redirect from the site's old path (the path you see in your browser, ie not the markdown .md filename (nor any Hugo files like _index.md)). 

Hugo uses the `aliases` attribute in this frontmatter section.

```
...
aliases:
    - /docs/reference/client 
```

See other examples: https://github.com/knative/docs/search?q=aliases&unscoped_q=aliases